Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 // Readme @ http://gitlab.iex.ec:30000/iexec/jenkins-library
-@Library('global-jenkins-library@1.6.1') _
+@Library('global-jenkins-library@1.6.2') _
```

Comment:
Do we update the local version [here](https://github.com/iExecBlockchainComputing/tee-worker-pre-compute/blob/develop/docker/sconify.sh#L9) or no need to?